internal/plugin/service/organization/address/zz_converters.go

```diff
@@ -0,0 +1,145 @@
+// Copyright (c) 2025 Aiven, Helsinki, Finland. https://aiven.io/
+// Code generated by user config generator. DO NOT EDIT.
+
+package address
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/hashicorp/terraform-plugin-framework/diag"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+
+	"github.com/aiven/terraform-provider-aiven/internal/plugin/util"
+)
+
+func idFields() []string {
+	return []string{"organization_id", "address_id"}
+}
+
+// dataModel Gets information about an organization address.
+type dataModel struct {
+	ID             types.String `tfsdk:"id"`
+	AddressID      types.String `tfsdk:"address_id"`
+	AddressLines   types.Set    `tfsdk:"address_lines"`
+	City           types.String `tfsdk:"city"`
+	CompanyName    types.String `tfsdk:"company_name"`
+	CountryCode    types.String `tfsdk:"country_code"`
+	CreateTime     types.String `tfsdk:"create_time"`
+	OrganizationID types.String `tfsdk:"organization_id"`
+	State          types.String `tfsdk:"state"`
+	UpdateTime     types.String `tfsdk:"update_time"`
+	ZipCode        types.String `tfsdk:"zip_code"`
+}
+
+type dtoModel struct {
+	AddressID      *string   `json:"address_id,omitempty"`
+	AddressLines   *[]string `json:"address_lines"`
+	City           *string   `json:"city"`
+	CompanyName    *string   `json:"company_name,omitempty"`
+	CountryCode    *string   `json:"country_code"`
+	CreateTime     *string   `json:"create_time,omitempty"`
+	OrganizationID *string   `json:"organization_id"`
+	State          *string   `json:"state,omitempty"`
+	UpdateTime     *string   `json:"update_time,omitempty"`
+	ZipCode        *string   `json:"zip_code,omitempty"`
```

Comment:
Each field is a pointer to handle all the required/optional cases as simple as `if v != nil`.